ztp/siteconfig-generator/siteConfig/siteConfigHelper.go

```diff
@@ -273,3 +336,49 @@ func mergeJsonCommonKey(mergeWith, mergeTo, key string) (string, error) {
 	}
 	return string(newJson), nil
 }
+
+func applyWorkloadPinningInstallConfigOverrides(clusterSpec *Clusters) (result string, err error) {
+	const (
+		deprecationCR      = "AgentClusterInstall"
+		deprecationField   = "cpuset"
+		deprecationMessage = "siteConfig.clusters[].node[].cpuset will be deprecated after OCP 4.15, please use siteConfig.clusters[].cpuPartitioningMode for OCP versions >= 4.13"
```

Comment:
better to have min message like "cpuset is deprecated instead use cpuPartitioningMode"

Reply:
I minimized it but left the version information since we won't know which version the user chooses, would that be okay?